README.md

```diff
@@ -1 +1,7 @@
-# zooms_classifier
\ No newline at end of file
+# ZooMS Classifier
+
+## Model Training and Explainability
+
+**Note:** Access to Google Drive is required to utilize this Colab notebook.
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1Ljos45IErs819W3ynY5-u9ach85y9J9G?usp=sharing)
```

Comment:
Can the notebooks be executed outside of Colab also? If so, can you document how to do so?

Reply:
done, see the README.md file